### PR TITLE
table-update

### PR DESCRIPTION
**Pull request by nazartokar**

### CODE REVIEW

**Comment by nazartokar** [comment]
This mixin is made to update table to use `flex` instead of `display: table`. It's needed to redesign headers correctly (`display: table` doesn't support `colspan`).

File: docs-json-parameters-table.html [hunk 1]
```diff
@@ -94,9 +94,7 @@ <h5 class="narrow-title">Properties</h5>
           <template is="dom-repeat" items="[[model]]">
             <template is="dom-if" if="[[item.isArrayHeader]]">
               <section class="tr">
-                <div class="td param-type"></div>
```
Comment (jarrodek) [inline docs-json-parameters-table.html:97]: Removing  this "cells" is causing the layout of the table header to collapse. Please, fix this before change.

**Review by jarrodek — CHANGES_REQUESTED** [review]
Removing

File: docs-json-parameters-table.html (hunk 1 above)
Comment (nazartokar) [reply docs-json-parameters-table.html:97]: @jarrodek ok, updated. Current mixins should affect unstyled state anyhow anymore. Only if they exist.

**Pull request merged**